docs/content/tutorials/ransomware.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Tutorial 2: Ransomware"
+date: 2020-05-26T20:57:10+03:00
+draft: false
+pre: '<i class="fab fa-graduation-cap"></i> '
+weight: 2
+tags: ["tutorials", "ransomware"]
+---
+
+In this tutorial, you will learn how to setup and run a ransomware scenario using Infection Monkey.
+This tutorial assumes that you've completed [Tutorial 1: Hello, Monkey](../hello-monkey), since it builds off of the knowledge gained from that tutorial.
+
+You'll learn:
+- How to setup an environment for a ransomware scenario
+- How to configure Infection Monkey to perform a ransomware attack
+- How to observe the results of the ransomware attack
+
+### Prerequisites
+First, make sure that you have the following installed:
+- `docker` and `docker-compose`
+
+### Run the environment
+Next, we'll use `docker compose` to run Infection Monkey along with our vulnerable container.
+
+1. Download the following compose file: [docker-compose.yml](../hello-monkey/docker/docker-compose.yaml)
+
+2. Then, navigate to the directory where you downloaded the file and run the following command to start the environment:
+
+   ```
+   docker compose up
+   ```
+
+Then, open a browser to [https://localhost:5000](https://localhost:5000), and login.
+
+### Configure the vulnerable container
+For this scenario, we're going to need some valuable data so that it can be held for ransom. We'll create a folder named `vault`, and we'll add a list of passwords to that vault.
+
+{{% notice warning %}}
+Infection Monkey will encrypt the contents of whichever folder we direct it to target. Therefore, when setting up a ransomware scenario, make sure you have a way to recover that data, or target a folder with dummy data.
```

Comment:
> whichever folder we direct it to target

This is perfectly correct language, but I feel like "direct" and "target" are _almost_ synonyms. If I didn't already know what you were talking about, this might throw me.